.travis.yml

```diff
@@ -1,12 +1,31 @@
 language: node_js
+
+env:
+  - TARGET=travis
+
 node_js:
-   - "0.10"
-   - "0.12"
-   - "4"
-   - "5"
-   - "6"
-   - "7"
-   - "8.4.0"
-   - "9"
-   - "10"
-script: "make travis"
+  - 0.10
+  - 0.12
```

Comment:
These will come out as floating point when the yaml is parsed. I'm not sure if that's a problem in practice, but it seems like most projects use strings for these versions numbers.

Reply:
It looks like it works, but maybe strings would be better 😜